app/views/candidate_mailer/new_referee_request.text.erb

```diff
@@ -1,17 +1,66 @@
-<% if @reason == :email_bounced %>
-Your referee request did not reach <%= @reference.name %>.
+<% if @application_form.show_new_reference_flow? %>
+  Dear <%= @application_form.first_name %>
 
-Check you gave the correct email address and request the reference again.
-<% elsif @reason == :refused %>
-<%= @reference.name %> has declined your reference request.
+  <% if @reason == :email_bounced %>
```

Comment:
This could be a tech debt issue for a future follow-up, but I wonder if we could split these conditions out into separate email templates, as they're not all that related?